server/src/test/java/io/crate/geo/GeoJSONUtilsTest.java

```diff
@@ -149,97 +146,106 @@ public void testMap2Shape() throws Exception {
             GeoJSONUtils.TYPE_FIELD, GeoJSONUtils.LINE_STRING,
             GeoJSONUtils.COORDINATES_FIELD, new Double[][]{{0.0, 0.1}, {1.0, 1.1}}
         ));
-        assertThat(shape, instanceOf(JtsGeometry.class));
-        assertThat(((JtsGeometry) shape).getGeom(), instanceOf(LineString.class));
+        assertThat(shape).isExactlyInstanceOf(JtsGeometry.class);
+        assertThat(((JtsGeometry) shape).getGeom()).isExactlyInstanceOf(LineString.class);
 
     }
 
     @Test
     public void testInvalidMap() throws Exception {
-        assertThrows(IllegalArgumentException.class,
-                     () -> GeoJSONUtils.map2Shape(Map.<String, Object>of()),
-                     "Cannot convert Map \"{}\" to shape");
```

Comment:
This argument was not the exception message, but the assert message. Therefore it is changed now.

Reply:
then please add:
`.hasMessage("Cannot convert GeoJSON \"{}\" to shape")`.

Reply:
apologies, ghub diff hid this for me.